ddl/db_integration_test.go

```diff
@@ -3413,3 +3413,13 @@ func (s *testIntegrationSuite3) TestTruncateLocalTemporaryTable(c *C) {
 	tk.MustExec("truncate table testt.t3")
 	tk.MustQuery("select * from testt.t3").Check(testkit.Rows())
 }
+
+func (s *testIntegrationSuite3) TestDropNoExitsColumn(c *C) {
```

Comment:
This test is not needed, since the error message is covered by other tests.

Reply:
That's right